### ISSUE TITLE
最新版混淆后无法正常播放，能否提供下混淆方法

### ISSUE THREAD

**Issue by MissFang** [penfeizhou/APNG4Android#108]
### New Issue Checklist

- [ ] I have searched for a similar issue in the [project](https://github.com/penfeizhou/APNG4Android/issues) and found none

### Issue Info

| Info                         | Value                                                 |      |
| ---------------------------- | ----------------------------------------------------- | ---- |
| Device Info                  | e.g. Huawei Mate 30 pro                               |      |
| System Version               | e.g.  6.0                                             |      |
| APNG4Android Library Version | e.g.  1.0.0                                           |      |
| Repro rate                   | e.g. all the time (100%) / sometimes x% / only once   |      |
| Repro with our demo project  | e.g. does it happen with our demo project?            |      |
| Demo project link            | e.g. link to a demo project that highlights the issue |      |

### Issue Description and Steps

Please fill in the detailed description of the issue (full output of any stack trace, compiler error, ...) and the steps to reproduce the issue.

**Comment by penfeizhou** [comment]
这个库里除了Gif播放,都可以混淆.
所以请看下是否是其他原因导致的无法播放.
请注意,Android的Drawable中的APNG图片经过AAPT编译后会破坏掉动画信息.

**Comment by MissFang** [comment]
（Android的Drawable中的APNG图片经过AAPT编译后会破坏掉动画信息.）那如果是这个原因应该怎么解决呢？

**Comment by penfeizhou** [comment]
APNG放到assets或raw中,或者关掉shrinkResources

**Comment by MissFang** [comment]
放到assets中就可以了，感谢大佬
